packages/Icon/src/index.ts

```diff
@@ -2,4 +2,5 @@ import createIconComponent from './createIconComponent';
 import glyphs from './glyphs';
 
 export { glyphs, createIconComponent };
+export { Size, IconProps } from './createIconComponent';
```

Comment:
Why exporting `IconProps`?

Reply:
You know... I'm sure I had a reason for this when I wrote it, but I can't think why it'd be any different than any other component now...

Reply:
Oh, well I found the reason pretty quick when I went to make it private. The `export default createIconComponent(glyphs)` line throws an error when `IconProps` is private:

> Default export of the module has or is using private name 'IconProps'

I suppose I could export it from `Icon.tsx` leave it out from `index.ts`, though.

Reply:
Explanation here: https://github.com/Microsoft/TypeScript/issues/6307#issuecomment-168321011